project/Build.scala

```diff
@@ -57,11 +57,8 @@ object Build extends Build {
     "-deprecation",
     "-language:_",
     "-Xlint",
-    "-Xlog-reflective-calls"
-  )
-
-  val fussyScalacOptions = basicScalacOptions ++ Seq(
-    "-Ywarn-unused",
+    "-Xlog-reflective-calls",
+        "-Ywarn-unused",
```

Comment:
Could you please fix the indenting here?
I usually leave these options off to speed up compilation but I see your point so I'll probably merge this PR.